### PR TITLE
Allow use of ${workspaceFolder} in python.interpreterPath

### PR DESCRIPTION
**Pull request by garymm**
Related to https://github.com/ansible/vscode-ansible/issues/900

### CODE REVIEW

**Comment by MLNW** [comment]
This would be great to have! Just spent at least an hour trying to set the interpreter through the `workspaceFolder` variable only to find issues stating that it is broken.

File: packages/ansible-language-server/src/utils/commandRunner.ts
Comment (priyamsahoo) [inline packages/ansible-language-server/src/utils/commandRunner.ts:36]:
```suggestion
    let interpreterPathFromConfig = this.settings.python.interpreterPath;
```

**Review by priyamsahoo — APPROVED** [review]
Looks good to me apart from a very minor suggestion.
The PR needs to wait for a while before we merge it.
Thanks.

**Review by ssbarnea — APPROVED** [review]

**Review by ssbarnea — APPROVED** [review]

**Comment by MLNW** [comment]
I see this was merged a while ago and since then plugin releases have been created. Yet when I use the ${workspaceFolder} variable in my settings file like this `"ansible.python.interpreterPath": "${workspaceFolder}/machines/ansible/.venv/bin/python",` and navigate to an Ansible file VS code says: 

image

If I click the warning I get this modal:

image

The recommended one is the correct one. Though if I select it the path in my VS code settings file is changed again to the full hard coded path. Not sure if this change just hasn't landed or it did not fully fix the issue.

Version of the plugin in VS code:

image

**Comment by TamiTakamiya** [comment]
@garymm If you switch to Pre-Release version, you can use `v24.9.1740521`, which contains #1397 changes.
image

**Pull request merged**
